ft/src/ft/resources/test-gradle-dms-plugin/settings.gradle.kts

```diff
@@ -1,6 +1,6 @@
 pluginManagement {
     plugins {
-        id("org.octopusden.octopus-dms-plugin") version (extra["dms-service.version"] as String)
```

Comment:
Why do we remove "-plugin" suffix?

Reply:
@pgorbachev, autogenerated artifactId is org.octopusden.octopus-dms.gradle.plugin (pluginId = org.octopusden.octopus-dms).

Also we don't use `plugin` suffix here https://github.com/octopusden/octopus-rm-gradle-plugin/blob/7a025e0233369afc58d7ac8806738404c69c5902/build.gradle#L37